### ISSUE TITLE
respect timefile/slack on missed execution

### ISSUE THREAD

**Issue by raisinrand** [leahneukirchen/snooze#21]
when running the following (to run a job every day if it has not run today)
```
snooze -s 1d -H 6 -t timefile -v
```
this happened
```
2023-11-22_00:20:54.57671 Snoozing until 2023-11-22T06:00:00-0500
2023-11-28_04:03:32.30171 Missed execution at 2023-11-22T06:00:00-0500
2023-11-28_04:03:32.30210 Snoozing until 2023-11-28T06:00:00-0500
```
it seems that if there is a missed execution, snooze will only look forward in time for the next execution.  however, if i were to run the original command again, snooze would not wait and instead exit immediately (because the timefile has not been touched today).

im not sure if this behavior is desired/intended, but it seems a little odd to me. sorry in advance if i'm missing something here

i think more intuitive behavior on missed execution would be to behave as if snooze were ran again, executing immediately if allowed by the combination of timefile/slack. would this change be welcome?

**Comment by Tumbleweeds** [comment]
I think the way it works is that since you missed the execution by 6 days and your slack is only 1 day, it will not try to execute again until midnight.

**Comment by raisinrand** [comment]
yeah, I think the weird thing is that I missed this execution by 6 days, but I also missed 6 other executions - one for each day - and in particular, one of them was less then a day ago. so I think it makes more sense to execute instead of waiting here
